Resolve conflicts with Fedora Chromium by using MD5-based BuildID.
https://bugzilla.rpmfusion.org/show_bug.cgi?id=5743

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -406,7 +406,7 @@ config("compiler") {
       # links.
       ldflags += [ "-Wl,--build-id=sha1" ]
     } else if (current_os != "aix" && current_os != "zos") {
-      ldflags += [ "-Wl,--build-id" ]
+      ldflags += [ "-Wl,--build-id=md5" ]
     }
 
     if (!is_android) {
